R/data.R

```diff
@@ -132,8 +132,11 @@
 #'   \item \code{prop_high_predation}: More details at \code{\link[DSMhabitat]{prop_high_predation}}
 #'   \item \code{contact_points}: More details at \code{\link[DSMhabitat]{contact_points}}
 #'   \item \code{..surv_juv_rear_int}: Intercept, Source: calibration (varies by tributary)
+#'   \item \code{.surv_juv_rear_contact_points}: Coefficient for \code{contact_points} variable, Source: TODO - Literature
```

Comment:
these changes need to make there way into the late fall run model

Reply:
compiling a list to change in late fall run